tests/integration/test_text_classification.py

```diff
@@ -81,31 +87,35 @@ def pipeline_dict() -> dict:
         },
     }
 
-    return pipeline_dict
+    return pipeline_dictionary
 
 
 @pytest.fixture
 def trainer_dict() -> dict:
+    """Returns the trainer dictionary"""
+
     return {
         "batch_size": 64,
         "num_epochs": 5,
         "optimizer": {"type": "adam", "lr": 0.01},
+        "cuda_device": -1,
     }
 
 
 def test_text_classification(
     tmp_path, pipeline_dict, trainer_dict, train_valid_data_source
```

Comment:
@ignacioct The CI of this PR fails because of this line, we renamed the fixture to `train_valid_dataset`. Could you push a small fix for this?

Reply:
it should be working now :)